code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1036,3 +1036,10 @@
 /datum/reagent/toxin/ambusher_toxin/on_mob_end_metabolize(mob/living/L)
 	L.remove_movespeed_modifier(type)
 	textShown = FALSE
+
+/datum/reagent/toxin/snakevenom
+	name = "Snake Venom"
+	description = "A potent venom extracted from a live snake"
+	color = "#792300" // rgb: 121, 35, 0
+	toxpwr = 2.5
```

Comment:
I agree with the toxin idea. There's in essence no difference between the two chemicals, and there's no other way to obtain it currently besides the limited supply of certain vendors. This would be an interesting way to produce this chemical.